packages/protocol/contracts/bridge/libs/LibBridgeRelease.sol

```diff
@@ -12,16 +12,20 @@ import { IBridge } from "../IBridge.sol";
 import { LibBridgeData } from "./LibBridgeData.sol";
 import { LibBridgeStatus } from "./LibBridgeStatus.sol";
 
+interface VaultContract {
```

Comment:
We should rely on [ERC-165](https://eips.ethereum.org/EIPS/eip-165) to determine if we should call `to.recallMessage`

Reply:
We cannot assume `to` contract is a vault. Vault is just one type of bridge apps.